### ISSUE TITLE
Docker errors for 1.5.2 and 1.5.3 Faktory server

### ISSUE THREAD

**Issue by gee-forr** [contribsys/faktory#370]
Hi there,

We were trying to use the x64 docker images from Dockerhub for Faktory server 1.5.2 & 1.5.3, but we were encountering the following error:

```
standard_init_linux.go:211: exec user process caused "exec format error"
```

We've currently pinned our Faktory server tag to 1.5.1 as that one is booting perfectly. I just thought I'd let you know as I've gone through the issues here, and it looks like this has happened once or twice before.

Thanks for Faktory @mperham, we ❤️ it big time.

**Comment by ThePletch** [comment]
Seeing the same issue, wanted to document attempted troubleshooting steps:

Since Fargate doesn't support ARM containers, I tried explicitly pointing my container definition at the 1.5.2 AMD64 digest - no dice. It looks like the architecture issue is affecting all available architectures for 1.5.2 (and 1.5.3 only has an ARM64 image on dockerhub, so it not working with fargate was sadly inevitable).

**Comment by mperham** [comment]
Thank you for bumping this issue, I'd lost it. I believe Faktory Enterprise's build has the fix, I just need to copy it in the OSS build.

**Comment by mperham** [comment]
Ok, I've tagged and pushed updated Docker images for 1.5.3/latest. Please let me know if they work for you.

**Comment by Konkrad** [comment]
Hello,
I had the same error and for me tag 1.5.3 is working now. Thank you for the fix.

**Comment by gee-forr** [comment]
This was fixed ages ago, working great in 1.6.0...

Thanks @mperham
